fiaas_deploy_daemon/config.py

```diff
@@ -224,12 +232,17 @@ def _parse_args(self, args):
         global_env_parser.add_argument("--global-env", default=[], env_var="FIAAS_GLOBAL_ENV",
                                        help="Various non-essential global variables to expose for all applications",
                                        action="append", type=KeyValue, dest="global_env")
+        global_datadog_tags_parser = parser.add_argument_group("Global Datadog tags", GLOBAL_DATADOG_TAGS_LONG_HELP)
+        global_datadog_tags_parser.add_argument("--global-datadog-tags", default=[], env_var="FIAAS_GLOBAL_DATADOG_TAGS",
```

Comment:
Maybe `datadog-global-tags` to be consistent with the other datadog-related config items?